openmetadata-ui/src/main/resources/ui/src/components/nav-bar/NavBar.tsx

```diff
@@ -241,28 +241,61 @@ const NavBar = ({
                 to={{
                   pathname: '/explore/tables',
                 }}>
-                {t('label.explore')}
+                {t('label.discover')}
               </NavLink>
-
               <NavLink
                 className="focus:tw-no-underline"
-                data-testid="appbar-item-glossary"
-                style={navStyle(pathname.startsWith('/glossary'))}
+                data-testid="appbar-item-data-quality"
+                style={navStyle(pathname.includes(ROUTES.TEST_SUITES))}
                 to={{
-                  pathname: ROUTES.GLOSSARY,
+                  pathname: ROUTES.TEST_SUITES,
                 }}>
-                {t('label.glossary')}
+                {t('label.data-quality')}
               </NavLink>
 
-              <NavLink
-                className="focus:tw-no-underline"
-                data-testid="appbar-item-tags"
-                style={navStyle(pathname.startsWith('/tags'))}
-                to={{
-                  pathname: ROUTES.TAGS,
-                }}>
-                {t('label.tags')}
-              </NavLink>
+              <Dropdown
+                className="cursor-pointer"
+                data-testid="governance"
+                overlay={
+                  <Menu
+                    items={[
+                      {
+                        key: 'glossary',
+                        label: (
+                          <NavLink
+                            className="focus:tw-no-underline"
+                            data-testid="appbar-item-glossary"
+                            style={navStyle(pathname.startsWith('/glossary'))}
+                            to={{
+                              pathname: ROUTES.GLOSSARY,
+                            }}>
+                            {t('label.glossary')}
+                          </NavLink>
+                        ),
+                      },
+                      {
+                        key: 'tags',
+                        label: (
+                          <NavLink
+                            className="focus:tw-no-underline"
+                            data-testid="appbar-item-tags"
+                            style={navStyle(pathname.startsWith('/tags'))}
+                            to={{
+                              pathname: ROUTES.TAGS,
+                            }}>
+                            {t('label.tags')}
+                          </NavLink>
+                        ),
+                      },
+                    ]}
```

Comment:
Pull it outside of the render to make render clean

Reply:
Sure.